appengine/standard/firebase/firetactoe/firetactoe.py

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/python2.4
+#
+# Copyright 2010 Google Inc. All Rights Reserved.
+
+# pylint: disable-msg=C6310
+
+"""Channel Tic Tac Toe
+
+This module demonstrates the App Engine Channel API by implementing a
+simple tic-tac-toe game.
+"""
+
+import datetime
```

Comment:
Imports should be in three sections: standard lib, third party, local in alphabetical order.